bin/cellranger/cellranger_count.sh

```diff
@@ -29,13 +30,19 @@ VERSION="$4"
 CPU="$5"
 MEM="$6"
 BAM_FLAG=""  # Default to generating BAM files
+CHEMISTRY="" # Default should detect chemistry
 REF="/software/cellgen/cellgeni/refdata_10x/refdata-gex-GRCh38-2024-A"
-
+echo "Arguments received: $@"
 # Handle optional --no-bam flag (disables BAM file generation)
 if [ "$7" == "--no-bam" ]; then
   BAM_FLAG="--no-bam"
 fi
 
+# Handle optional --chemistry flag (default is not needed)
```

Comment:
This won't work is the above `--no-bam` flag isn't set. 

You did some nice work in the `cellranger.sh` script, to parse optional arguments. Use the same code here!